docs/symptom-survey/coding.md

```diff
@@ -305,6 +310,14 @@ survey, even if they have already taken it before:
 > have participated before. Completing the survey again will help us understand
 > how the situation is changing.
 
+It also more clearly states that the data may be shared and aggregates publicly
+released:
```

Comment:
```suggestion
Wave 5 now clearly states that the data may be shared and aggregates publicly when
released:
```

I'm not entirely sure what you wanted to say here, but the sentence felt choppy.